crd/multitenancy/manifests/multitenancy.acn.azure.com_multitenantpodnetworkconfigs.yaml

```diff
@@ -119,6 +123,8 @@ spec:
               primaryIP:
                 description: Deprecated - use InterfaceInfos
                 type: string
+            required:
+            - defaultDenyACL
```

Comment:
once this field is marked optional it should be dropped from the list of required fields indicated here

Reply:
removed it, thx